### ISSUE TITLE
high severity vulnerabilities on fresh install of `nextra-theme-blog@alpha`

### ISSUE THREAD

**Issue by ek** [shuding/nextra#3827]
**Describe the bug** A clear and concise description of what the bug is.

6 high severity vulnerabilities on fresh install into empty directory

**To Reproduce** Steps to reproduce the behavior:

1. `npm i nextra-theme-blog@alpha`
2. `npm audit fix --force`

**Expected behavior** A clear and concise description of what you expected to
happen.

Expected to install nextra alpha with blog theme.

**Screenshots** If applicable, add screenshots to help explain your problem.

after running `npm audit fix --force`:

```js
npm warn using --force Recommended protections disabled.

up to date, audited 447 packages in 1s

174 packages are looking for funding
  run `npm fund` for details

# npm audit report

cross-spawn  <6.0.6
Severity: high
Regular Expression Denial of Service (ReDoS) in cross-spawn - https://github.com/advisories/GHSA-3xgq-45jj-v275
fix available via `npm audit fix`
node_modules/cross-spawn
  execa  0.5.0 - 0.9.0
  Depends on vulnerable versions of cross-spawn
  node_modules/execa
    clipboardy  <=1.2.3
    Depends on vulnerable versions of execa
    node_modules/clipboardy
      title  3.0.0 - 3.5.3
      Depends on vulnerable versions of clipboardy
      node_modules/title
        nextra  2.0.0-alpha.0 - 3.2.3 || 4.0.0-app-router.0 - 4.0.0-app-router.27
        Depends on vulnerable versions of title
        node_modules/nextra
          nextra-theme-blog  2.2.12 - 3.2.3 || 4.0.0-app-router.0 - 4.0.0-app-router.27
          Depends on vulnerable versions of nextra
          node_modules/nextra-theme-blog

6 high severity vulnerabilities

To address all issues, run:
  npm audit fix
```

**Desktop (please complete the following information):**

- OS: MacOS 15.1.1 (24B91)
- NPM version 10.9.1

**Additional context** Add any other context about the problem here.

**Comment by dimaMachina** [comment]
please precise nextra version installed

**Comment by dimaMachina** [comment]
`nextra-theme-blog@alpha` is resolved to `"nextra-theme-blog": "^3.0.0-alpha.42"` which was released

this issue is fixed in https://github.com/shuding/nextra/issues/3674
